src/oop/app/src/Transporters/GuzzleAdapter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace src\oop\app\src\Transporters;
+
+use src\oop\app\src\Transporters\TransportInterface;
+use \GuzzleHttp\Client;
+
+class GuzzleAdapter implements TransportInterface
+{
+    private Client $client;
+
+    public function __construct()
+    {
+        $this->client = new Client();
```

Comment:
please inject Client instead of initialization.

Reply:
done